### PR TITLE
DOC: notes on simplifying assumptions in prl sim

### PR DESCRIPTION
**Pull request by aflaxman**

### CODE REVIEW

**Review by zmbc — APPROVED** [review]
LGTM; I do not understand the unemployment problem but it seems like it would take me some time to figure out what it means.

**Review by beatrixkh — APPROVED** [review]

File: docs/source/concept_models/vivarium_census_synthdata/concept_model.rst [hunk 1]
```diff
@@ -1026,6 +1035,15 @@ addresses for businesses and households, but eventually we might want
...
 business out of their home.
 
+To keep things simple, for now when businesses move to a new address,
+it will be a totally new address. No household or business will ever
+move into their old address.
+
+We have also included a special "employer" to indicate individuals who
+are *not* currently employed.  We assume that 58% of the population is
+employed, which leads to a lot of individual switching to being
+unemployed.  We might need to refine this in the future.
```
Comment (beatrixkh) [inline docs/source/concept_models/vivarium_census_synthdata/concept_model.rst:1045]: right now i have that 42% of adults are initialized as unemployed, and the remaining 58% have employers whose sizes fit a lognormal distribution. then people switch to a new job selected uniformly at random. people are steadily becoming increasingly unemployed, and employer sizes are steadily becoming uniform

image

File: docs/source/concept_models/vivarium_census_synthdata/concept_model.rst [hunk 2]
```diff
@@ -332,6 +336,47 @@ mark the last 1-17 people as untracked -- which isn't so much memory
...
+whether it is Institutional or Non-institutional GQ (in the TYPE
+variable: 2 = Institutional; 3 = Non-institutional).
+
+The third and final step for initializing GQ simulants is to give each
+a (somewhat inappropriately named) household_id.  Eventually we shall
+accomplish this so that the distribution of GQ sizes match what is
+found in census, but as a simple stand-in for now we will include 6
+special "household_id" values for the six broad types of GQs that we
+wish to represent, and assign simulants to one of the categories
+consistent with their GQ TYPE uniformly at random.  The GQ subtypes of
```
Comment (beatrixkh) [inline docs/source/concept_models/vivarium_census_synthdata/concept_model.rst:374]: when we eventually add complexity to this, some constraints might include
- age-dependent probability of college
- age-dependent probability of nursing home
- (other gq types also prob age-dependent, but these ^ perhaps most obviously so?)
- anyone "retired" unlikely to move into military GQ

**Review by beatrixkh — APPROVED** [review]
for future, will want strategy for how people move out of GQ

**Pull request merged**